Error while checking build/O2/fullCI for 69fb6b45418d675a0204370b89468051660b1cb6 at 2023-06-22 20:14:
```
## sw/BUILD/O2-latest/log
/sw/SOURCES/O2/11545-slc8_x86-64/0/Detectors/ITSMFT/ITS/tracking/GPU/cuda/TrackerTraitsGPU.cu(434): error: no instance of constructor "o2::its::Cell::Cell" matches the argument list
ninja: build stopped: subcommand failed.
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/AliceO2/11545/69fb6b45418d675a0204370b89468051660b1cb6/build_O2_fullCI/pretty.html).

--- a/Detectors/ITSMFT/ITS/tracking/GPU/cuda/TrackerTraitsGPU.cu
+++ b/Detectors/ITSMFT/ITS/tracking/GPU/cuda/TrackerTraitsGPU.cu
@@ -434,8 +434,7 @@ GPUg() void computeLayerCellsKernel(
           new (cells + cellsLUT[iCurrentTrackletIndex] + foundCells) Cell{currentTracklet.firstClusterIndex, nextTracklet.firstClusterIndex,
                                                                           nextTracklet.secondClusterIndex,
                                                                           iCurrentTrackletIndex,
-                                                                          iNextTrackletIndex,
-                                                                          tanLambda};
+                                                                          iNextTrackletIndex};
         }
         ++foundCells;
       }

